apps/blockchain/test/blockchain/transaction/receipt/bloom_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Blockchain.Transaction.Receipt.BloomTest do
+  use ExUnit.Case
+  alias Blockchain.Transaction.Receipt.Bloom
+
+  describe "contains?/2" do
+    test "adds value to new filter and checks if it is in filter" do
```

Comment:
Should this just say `"checks if value is in filter"` since it's a test under `contains?/2` or does the function also add a value to the bloom filter?